src/components/Analytics/PageviewTracker.tsx

```diff
@@ -45,8 +44,11 @@ function usePageTracking() {
   useEffect(() => {
     if (initialized) {
       // We only add the legacy tracker explicitly here, because by default, GA
-      // will send always send hits to the default tracker.
-      ReactGA.pageview(pathname, [legacyTracker.name]);
+      // will send always send hits to the default tracker. We add a delay to
+      // give the app time to update the page title.
+      setTimeout(() => {
+        ReactGA.pageview(pathname, [legacyTracker.name], document.title);
+      }, 200);
```

Comment:
FWIW-  This makes me a tiny bit nervous... e.g. people that bounce off the page might be less likely to be counted as a pageview now...  If you think of it, it would be good to sanity check the page views in GA after a few days to make sure it looks stable.

Reply:
I tested closing the tab immediately after navigating to a page and we don't capture the page view - I'm going to try something else, good catch!